infrastructure/main.tf

```diff
@@ -11,28 +11,57 @@ variable s3_bucket_name {
   default = "staging.react.london"
 }
 
-resource "aws_s3_bucket" "staging" {
+resource "aws_s3_bucket" "website" {
   provider = "aws"
 
   bucket = "${var.s3_bucket_name}"
   acl = "public-read"
   policy = <<POLICY
-{
-	"Version": "2012-10-17",
-	"Statement": [
-		{
-			"Sid": "PublicReadGetObject",
-			"Effect": "Allow",
-			"Principal": "*",
-			"Action": "s3:GetObject",
-			"Resource": "arn:aws:s3:::${var.s3_bucket_name}/*"
-		}
-	]
-}
-POLICY
+  {
+    "Version": "2012-10-17",
+    "Statement": [
+      {
+	"Sid": "PublicReadGetObject",
+	"Effect": "Allow",
+	"Principal": "*",
+	"Action": "s3:GetObject",
+	"Resource": "arn:aws:s3:::${var.s3_bucket_name}/*"
+      }
+    ]
+  }
+  POLICY
 
   website {
     index_document = "index.html"
     error_document = "index.html" # Handle 404 errors in JS app
```

Comment:
Nah, `error_document` is the file served when your request 404s. We should probably render a 404.html when we have the ability to render multiple pages.